zerver/lib/markdown/__init__.py

```diff
@@ -1519,7 +1519,7 @@ def handleMatch(self, match: Match[str]) -> Optional[Union[str, Element]]:
             return make_realm_emoji(active_realm_emoji[name]["source_url"], orig_syntax)
         elif name == "zulip":
             return make_realm_emoji(
-                "/static/generated/emoji/images/emoji/unicode/zulip.png", orig_syntax
+                settings.STATIC_URL + "generated/emoji/images/emoji/unicode/zulip.png", orig_syntax
```

Comment:
Stylistically, I think I might prefer the f-strings version:
`f"{settings.STATIC_URL}generated/emoji/images/emoji/unicode/zulip.png"` if we're not doing something like `urllib.parse.urljoin(settings.STATIC_URL, "generated/...")`.